### PR TITLE
Fix vita elf vaddr and size

### PR DESCRIPTION
**Pull request by Princess-of-Sleeping**

### CODE REVIEW

**Review by d3m3vilurr — APPROVED** [review]
@frangarcj could you review this?

File: patches/binutils/0003-fix-elf-vaddr.patch [hunk 1]
```diff
@@ -0,0 +1,12 @@
+diff -Nru a/src/binutils/ld/emulparams/armelf.sh b/src/binutils/ld/emulparams/armelf.sh
+--- a/src/binutils/ld/emulparams/armelf.sh
++++ b/src/binutils/ld/emulparams/armelf.sh
+@@ -3,7 +3,7 @@
+ OUTPUT_FORMAT="elf32-littlearm"
+ BIG_OUTPUT_FORMAT="elf32-bigarm"
+ LITTLE_OUTPUT_FORMAT="elf32-littlearm"
+-TEXT_START_ADDR=0x8000
++TEXT_START_ADDR=0x81000000
```
Comment (yne) [inline patches/binutils/0003-fix-elf-vaddr.patch:9]: Why `0x81000000` ?

Comment (Princess-of-Sleeping) [reply patches/binutils/0003-fix-elf-vaddr.patch:9]: The base address of most eboot.bin and self, skprx, suprx of vita is 0x81000000.
image

Comment (d3m3vilurr) [reply patches/binutils/0003-fix-elf-vaddr.patch:9]: @yne is it an answer for you and do you have more question?

**Comment by bythos14** [comment]
C++ binaries built with this patch cause vita-elf-create to segfault, the exact reason is unknown

File: patches/binutils/0003-fix-elf-vaddr.patch (hunk 1 above)
Comment (yne) [reply patches/binutils/0003-fix-elf-vaddr.patch:9]: sorry for the delay,
still unsure about the necessity of this changes.
I hope it does not interfere with any previous tools.

**Comment by Princess-of-Sleeping** [comment]
Crash location is `vita-elf-create(vita-elf-create.c) -> vita_elf_load(vita-elf.c) -> maybe https://github.com/vitasdk/vita-toolchain/blob/master/src/vita-elf.c#L463-L464 -> maybe load_rel_table -> ???`

**Pull request merged**